### PR TITLE
Or 1709 thanos update super chain logic

### PR DESCRIPTION
**Pull request by nguyenzung**
Apply super chain config for L1CrossDomainMessenger and L1StandardBridge

Thank you very much!

### CODE REVIEW

File: packages/tokamak/contracts-bedrock/src/L1/L1CrossDomainMessenger.sol
Comment (boohyung) [inline packages/tokamak/contracts-bedrock/src/L1/L1CrossDomainMessenger.sol:1]: Now we have already used require for paused() in [CrossDomainMessenger.sol](https://github.com/tokamak-network/tokamak-thanos/blob/main/packages/tokamak/contracts-bedrock/src/universal/CrossDomainMessenger.sol#L221). Can you share why it should be checked here?

Comment (nguyenzung) [reply packages/tokamak/contracts-bedrock/src/L1/L1CrossDomainMessenger.sol:1]: Thank you, @boohyung 
The function relayMessage is overrided in L1CrossDomainMessenger so the using of paused in base CrossDomainMessenger.relayMessage does not have any meaning on the L1

**Review by boohyung — APPROVED** [review]
Thank you so much, Brave!

**Comment by rlgns98kr** [comment]
Can we update the slither version to `0.4.0` ?
https://github.com/crytic/slither-action

**Pull request merged**